app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,30 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
+
+  def show
+    @post = Post.find(params[:id])
+    @user = @post.user
+    @comments = @post.comments
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @new_post = current_user.posts.new(params.require(:post).permit(:title, :text))
+    @new_post.likes_counter = 0
+    @new_post.comments_counter = 0
+    @new_post.update_posts_counter
+    respond_to do |_format|
+      if @new_post.save
+        redirect_to "/users/#{@new_post.user.id}/posts/", notice: 'Created Successfuly'
+      else
```

Comment:
- [ ] (Optional) When I create a new post, it doesn't redirect me to the post index page. It would be nice to implement this as a good UI feature.

![chrome-capture (4)](https://user-images.githubusercontent.com/77106962/163686341-ed13cfd2-f366-4eda-bc50-248a29eb5697.gif)